includes/Admin/Dashboard.php

```diff
@@ -573,4 +574,32 @@ public function display_link_to_dashboard(): void {
 		</div>
 		<?php
 	}
+
+	/**
+	 * Load the Speculation Rules for the Dashboard.
+	 *
+	 * @return array An array containing prerendering rules.
+	 */
+	public function load_speculation_rules(): array {
```

Comment:
Turns out this code is still running on the frontend. Apparently the `Dashboard` class is still instantiated on the frontend too.

To fix this:

1. Make the class implement the `Conditional` interface
2. Add the following class method

```php
	/**
	 * Check whether the conditional object is currently needed.
	 *
	 * @since 1.37.0
	 *
	 * @return bool Whether the conditional object is needed.
	 */
	public static function is_needed(): bool {
		return is_admin();
	}
```

This way the `Dashboard` class is truly only instantiated in the admin.

Not a fault of this PR, this was an existing issue.